stable/redis/values-production.yaml

```diff
@@ -0,0 +1,230 @@
+## Bitnami Redis image version
+## ref: https://hub.docker.com/r/bitnami/redis/tags/
+##
+image: bitnami/redis
+imageTag: 4.0.9-r0
+
+## Specify a imagePullPolicy
+## ref: http://kubernetes.io/docs/user-guide/images/#pre-pulling-images
+##
+imagePullPolicy: IfNotPresent
+
+## Image credentials
+# imageCredentials:
+  # registry: docker.io
+  # username: javsalgar
+  # password: password
+
+## Cluster settings
+cluster:
+  enabled: true
+  slaveCount: 3
+
+## Use existing secret (ignores previous password)
+# existingSecret:
+
+## Enable persistence using Persistent Volume Claims
+## ref: http://kubernetes.io/docs/user-guide/persistent-volumes/
+##
+persistence:
```

Comment:
Should be this part of the "Master" parameters?

Reply:
I was not sure if we should add another level inside the master section.

Reply:
@sameersbn @tompizmor thoughts?

Reply:
makes sense to move it to `master` parameters.

On a related note, why don't we move the slave object to a StatefulSet and add persistence for the slaves as well? Do you think there will be any benefit of persisting the slave data?